client/src/pages/MyGarden.jsx

```diff
@@ -0,0 +1,110 @@
+import { DELETE_PLANT } from "../utils/mutations";
+
+const fakePlants = Array(5)
+  .fill(null)
+  .map(() => ({
+    name: "plant",
+    _id: crypto.getRandomValues(new Uint32Array(1))[0],
+    image: "https://via.placeholder.com/150/flower",
+  }));
+
+/**
+ *
+ * @param {*} props
+ * @returns
+ */
+function MyGarden(props) {
+  const auth = useAuthService();
+  const { data, error, loading } = useQuery(QUERY_PLANTS);
+  const [deleteMutation] = useMutation(DELETE_PLANT, {
+    refetchQueries: [{ query: QUERY_PLANTS }],
+    onError: (e) => {
+      /* TO DO: add error handling */
+      console.error(e.message);
+      alert("Error deleting plant: " + e.message);
+    },
+  });
+
+  if (loading) {
+    return <CenteredCircularProgress />;
+  }
+
+  const [firstPlant, ...restOfPlants] = data?.plants;
+
+  const deletePlant = (plant, e) => {
+    e.preventDefault();
+    console.log("delete plant", plant);
+    deleteMutation({ variables: { id: plant._id } }).then();
+  };
+
+  return (
+    <Stack className="MyGarden" sx={{ direction: "row", placeItems: "center" }}>
+      <Stack sx={{ p: 5, width: "clamp(300px,60%,700px)" }}>
+        <Box className="myGardenHeader">
+          <p>Hello</p>
+          <Typography variant="h1">{auth.user.username}</Typography>
+        </Box>
+
+        {/* plants list in a 2/2 grid */}
+        <Box
+          sx={{
+            display: "grid",
+            gridTemplateColumns: "1fr 1fr",
+            gridTemplateRows: "200px",
+            gap: 2,
+          }}
+        >
+          {/* first plant takes 2 columns and 2 rows*/}
+          <Box sx={{ gridColumn: "1 / 3", gridRow: "1 / 3", maxHeight: "400px", position: "relative" }}>
+            <NavLink to={`/plants/${firstPlant._id}`}>
+              <ResponsiveImageContainer image={firstPlant.img} />
+              <FloatViewDetails />
+              <IconButton
+                sx={{ position: "absolute", right: 2, top: 2, color: "white", backgroundColor: "grey" }}
+                onClick={(e) => deletePlant(firstPlant, e)}
+              >
+                <DeleteRounded />
+              </IconButton>
+            </NavLink>
+          </Box>
+
+          {/* rest of the plants take 1 column and 1 row */}
+          {restOfPlants.map((plant) => (
+            <Box sx={{ position: "relative" }}>
+              <NavLink to={`/plants/${plant._id}`} key={plant._id}>
+                <ResponsiveImageContainer key={plant._id} image={plant.img} />
+                <FloatViewDetails />
+                <IconButton
+                  sx={{ position: "absolute", right: 2, top: 2, color: "white", backgroundColor: "grey" }}
+                  onClick={(e) => deletePlant(plant, e)}
+                >
+                  <DeleteRounded />
+                </IconButton>
+              </NavLink>
+            </Box>
+          ))}
+        </Box>
+      </Stack>
+    </Stack>
+  );
+}
+
+export default MyGarden;
+
+function FloatViewDetails() {
```

Comment:
Should be its own component
This doesn't match the figma mock.
This should be in the single plant/plant detail view.